libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -81,6 +81,15 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     math::mat4f userWorldFromWorldMatrix;   // userWorld <- world
     math::float4 clipTransform;             // [sx, sy, tx, ty] only used by VERTEX_DOMAIN_DEVICE
 
+    // --------------------------------------------------------------------------------------------
+    // Stereoscopic rendering
+    // --------------------------------------------------------------------------------------------
+
+    math::mat4f eyeFromWorldMatrix[4];
+    math::mat4f clipFromEyeMatrix[4];
```

Comment:
why 4 matrices, it seems like the filament side API is limited to 2?

Reply:
Yup, I think this was a typo on my end.